### PR TITLE
Fix responsivity on dataset details page

### LINKED ISSUE #205: Details page is not readable on a mobile
It displays like this:
image

### PR DESCRIPTION
**Pull request by l-vincent-l**
Was
image
Now
image

fix #205

### CODE REVIEW

File: client/stylesheets/components/_dataset-details.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
-.dataset-details__side-pan {
-  padding-right: 2em;
-  width: 20%;
+.dataset-details__side-pane {
```
Comment (maukoquiroga) [inline client/stylesheets/components/_dataset-details.scss]: panel ?

File: client/stylesheets/components/_dataset-details.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+  background: $background-white !important;
```
Comment (maukoquiroga) [inline client/stylesheets/components/_dataset-details.scss]: Oops, try not to use ```!important```.

File: client/stylesheets/components/_dataset-details.scss [hunk 3]
```diff
@@ -57,7 +53,7 @@
       border-right: 0;
     }
 
-    span:last-child {
+    spane:last-child {
```
Comment (maukoquiroga) [inline client/stylesheets/components/_dataset-details.scss]: ?

File: lib/transport_web/templates/dataset/details.html.eex [hunk 4]
```diff
@@ -4,7 +4,7 @@
   <% end %>
   <h1><%= @dataset.title %></h1>
   <div class="documentation">
-    <div class="side-pan dataset-details__side-pan">
+    <div class="side-pane dataset-details__side-pane">
```
Comment (maukoquiroga) [inline lib/transport_web/templates/dataset/details.html.eex]: panel?

Comment (l-vincent-l) [reply lib/transport_web/templates/dataset/details.html.eex]: nop

**Pull request merged**